lib/src/components/NxNavigationSidebar/NxNavigationSidebar.scss

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+@import '../../scss-shared/nx-container-helpers';
+
+$nx-sidebar-max-width: 300px;
+$nx-sidebar-width__open: 250px;
```

Comment:
I haven't seen us use BEM syntax in variable names before.  @glenhunter what do you think of that?

Reply:
> I haven't seen us use BEM syntax in variable names before. @glenhunter what do you think of that?

We have tended to avoid using BEM for variables. I think this is fine as: `$nx-sidebar-width-open`.

After my PR to convert the page with variables to CSS `var` from SCSS `$` is merged we'll likely want to convert these as well.

FWIW the BEM naming about is wrong anyway, it would be `$nx-sidebar-width--open` - a modifier of `$nx-sidebar-width`

Reply:
> > I haven't seen us use BEM syntax in variable names before. @glenhunter what do you think of that?
> 
> We have tended to avoid using BEM for variables. I think this is fine as: `$nx-sidebar-width-open`.
> 
> After my PR to convert the page with variables to CSS `var` from SCSS `$` is merged we'll likely want to convert these as well.
> 
> FWIW the BEM naming above is wrong anyway, it would be `$nx-sidebar-width--open` - a modifier of `$nx-sidebar-width`

Fixed in 882c4209a1e9870b0fd02a20d4e7e7d8c8be58b2